source/manual/deploying-terraform.html.md

```diff
@@ -23,65 +23,77 @@ to our AWS environments.
 
 You can find which class of user you are [infra-security project in govuk-aws-data](https://github.com/alphagov/govuk-aws-data/tree/master/data/infra-security).
 
-## Deploying manually on Jenkins
+### 2. Install `gds-cli`
 
-> **This is not the recommended approach.**
->
-> It's slightly easier to start with, but you'll quickly want to switch to using the `deploy.rb` helper script (see next section), to avoid pasting secrets over and again!
+[`gds-cli`](/manual/access-aws-console.html) is the preferred way of obtaining
+AWS credentials.
 
-### 1. Generate an AWS token and key
+As of version `v2.15.0` of [`gds-cli`](/manual/access-aws-console.html),
+you can use it to deploy terraform via Jenkins.
 
-First make sure you have [`gds-cli`](/manual/access-aws-console.html) installed.
+### 3. Get Github Credentials
 
-👉 [Instructions for installing `gds-cli`](/manual/access-aws-console.html)
+You need to obtain your Github credentials by creating a read-only GitHub personal access token. This [GitHub personal access token](https://github.com/settings/tokens) should be
+created with the `read:org` scope only.
 
-Run the following to generate a token and key.
-
-```
-gds aws <your-role e.g. govuk-integration-poweruser> -e
-```
+> Take care to store and handle the token securely. If you accidentally share your token,
+  [revoke it immediately](https://github.com/settings/tokens) and follow the
+  [instructions for reporting a potential data security incident][security-incidents].
 
-### 2. Run the Jenkins job
+## Deploying Terraform
 
 **Always `plan` first, check that the output is what you expect, then `apply`.**
 
-Manually run the Jenkins job, pasting the token and key from the previous step.
+There are 2 ways of deploying terraform:
 
-👉 [Deploy terraform directly from Jenkins][ci-deploy-jenkins]
+1. [`gds-cli`](/manual/access-aws-console.html) as of version `v2.15.0`
+2. `deploy.rb` script in [govuk-aws][deploy-rb] repository
 
-## Deploying with `deploy.rb`
+### 1. [`gds-cli`](/manual/access-aws-console.html) as of version `v2.15.0`
```

Comment:
I don't think we need to repeat the version and link throughout - this would do:

```suggestion
### 1. gds-cli
```